analyses/interaction-plots/scripts/02-process_mutations.R

```diff
@@ -137,27 +175,28 @@ if(exists("specimen_file")){
 
 ### Reduce MAF to a smaller set of relevant columns
 
-maf_df <- maf_df %>% 
-  dplyr::select(Hugo_Symbol, 
-                Entrez_Gene_Id, 
-                Chromosome, 
-                Start_Position,
-                End_Position, 
-                Strand, 
-                Variant_Classification,
-                Variant_Type,
-                Reference_Allele,
-                Tumor_Seq_Allele1,
-                Tumor_Seq_Allele2,
-                Tumor_Sample_Barcode,
-                t_depth, 
-                t_ref_count,
-                t_alt_count,
-                Consequence,
-  )  
+maf_df <- maf_df %>%
+  dplyr::select(
+    Hugo_Symbol,
+    Entrez_Gene_Id,
+    Chromosome,
+    Start_Position,
+    End_Position,
+    Strand,
+    Variant_Classification,
+    Variant_Type,
+    Reference_Allele,
+    Tumor_Seq_Allele1,
+    Tumor_Seq_Allele2,
+    Tumor_Sample_Barcode,
+    t_depth,
+    t_ref_count,
+    t_alt_count,
+    Consequence,
```

Comment:
Does the trailing `,` not cause issues here?

Reply:
apparently not? I'll kill it anyway.